css/app.css

```diff
@@ -562,3 +618,12 @@ input[type=search]:focus {
 header.search-active ~ #editor div.CodeMirror .cm-matchhighlight {
   background-color: rgba(253, 207, 76, .5);
 }
+
+.hidden {
+  position: absolute;
+  bottom: -999999;
```

Comment:
I copied this css to match what codemirror does on the assumption that whatever it chose would be the most stable, in retrospect we can probably use visibility:hidden because we don't have to support IE